pkg/parser/digester.go

```diff
@@ -145,6 +145,9 @@ type sqlDigester struct {
 	lexer  *Scanner
 	hasher hash2.Hash
 	tokens tokenDeque
+	// when mark==true, content will not be replaced by '?'
+	// but it we wrapped by unicode `<>`
+	marker bool
```

Comment:
What is this?

Reply:
Pulled changes from another branch. Deleted.